packages/button/index.js

```diff
@@ -16,7 +17,7 @@ import { messages as svMessages } from './locales/sv/messages.mjs';
 
 const buttonTypes = ['primary', 'secondary', 'negative', 'utility', 'pill', 'link'];
 
-class WarpButton extends kebabCaseAttributes(WarpElement) {
+class WarpButton extends FormControlMixin(kebabCaseAttributes(WarpElement)) {
```

Comment:
Add the open wc mixin so we dont need to implement our own. This provides some hooks and a `this.setValue` method for us and sets up the element internals under the hood